docs/konflux/release_operator_with_konflux.md

```diff
@@ -396,10 +396,10 @@ schema: olm.bundle
   ...
   ...
   fbc-v4-14-5p7m9	True	Merge pull request #81 from jordigilh/release/1.2.0-rc6
-  fbc-v4-14-jv6f8	True	Merge pull request #83 from parodos-dev/konflux/references/main
-  fbc-v4-14-dhxqb	True	Merge pull request #82 from parodos-dev/konflux/component-updates/operator-bundle
-  fbc-v4-14-bdx8p	True	Merge pull request #85 from parodos-dev/konflux/component-updates/operator-bundle
-  fbc-v4-14-hftq5	True	Merge pull request #84 from parodos-dev/konflux/references/main
+  fbc-v4-14-jv6f8	True	Merge pull request #83 from rhdhorchestrator/konflux/references/main
```

Comment:
@jenniferubah @jordigilh same here

Reply:
I think it will be fine once the MR I raised is merged.